CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# How to Contribute
+
+Contributions are essential for keeping this language server great. We try to keep it as easy as possible to contribute changes and we are open to suggestions for making it even easier. There are only a few guidelines that we need contributors to follow.
+
+## Development
+
+### Installation Prerequisites:
+
+  * [JDK 8+](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+  * [Maven](https://maven.apache.org/)
+
+### Steps
+1. Fork and clone this repository
+
+2. Build/Test LemMinx on Mac/Linux:
+	```bash
+	$ ./mvnw verify
+	```
+	or for Windows:
+	```bash
+	$ mvnw.cmd verify
+	```
+
+### Debug
+
```

Comment:
I think you should explicitly mention that you need to connect to the language server through remote debugging.

Reply:
Done.